nltk/corpus/reader/verbnet.py

```diff
@@ -172,9 +172,77 @@ def fileids(self, vnclass_ids=None):
             return [self._class_to_fileid[self.longid(vnclass_id)]
                     for vnclass_id in vnclass_ids]
 
+    def frames(self, vnclass):
+        """
+        Given a verbnet class, this method returns frames with members-
```

Comment:
I would rephrase this to follow https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings
```python
def frames(self, vnclass):
    """Return frames with members.

    The members are:
    1) Example
    ...

"""